ibis/backends/impala/compiler.py

```diff
@@ -17,7 +22,7 @@ def _get_join_type(self, op):
 
 
 class ImpalaExprTranslator(ExprTranslator):
-    _registry = {**operation_registry, **binary_infix_ops}
+    _registry = {**operation_registry, **binary_infix_ops, **{ops.Hash: hash}}
```

Comment:
Could probably even do `**{ops.Hash: unary("fnv_hash")}` so you can kill the `hash` function entirely.